lte/gateway/c/core/oai/lib/n11/amf_client_proto_msg_to_itti_msg.cpp

```diff
@@ -85,4 +85,19 @@ void convert_proto_msg_to_itti_m5g_auth_info_ans(
   return;
 }
 
+void convert_proto_msg_to_itti_amf_decrypted_imsi_info_ans(
+    M5GSUCIRegistrationAnswer response,
+    itti_amf_decrypted_imsi_info_ans_t* amf_app_decrypted_imsi_info_resp) {
+  if (response.ue_msin_recv().length() <= 0) {
+    std::cout << "[ERROR] Decrypted IMSI response is invalid:"
+              << response.ue_msin_recv().length() << std::endl;
```

Comment:
use OAI LOG

Reply:
These GRPC Response ERROR messages are directed to syslog using standard std::cout to indicate the failures. Do you want  me to localize this to mme log. Let me know your inputs.